test/integration-tests/V3ToV4Migration.test.ts

```diff
@@ -1032,8 +1032,8 @@ describe('V3 to V4 Migration Tests:', () => {
         '0x',
       ])
 
-      v4Planner.addAction(Actions.SETTLE_WITH_BALANCE, [USDC.address])
-      v4Planner.addAction(Actions.SETTLE_WITH_BALANCE, [WETH.address])
+      v4Planner.addAction(Actions.SETTLE, [USDC.address, 1, false])
+      v4Planner.addAction(Actions.SETTLE, [WETH.address, 1, false])
```

Comment:
Then these tests can have these constants that are already in constants.ts
1 -> `CONTRACT_BALANCE`
false -> `SOURCE_ROUTER`

Reply:
since CONTRACT_BALANCE and OEPN_DELTA now switched, i used OPEN_DELTA instead

Reply:
wait what

Reply:
OH yeah youre right